contracts/contracts.go

```diff
@@ -53,6 +57,10 @@ func init() {
 	IERC721 = artifact("IERC721", tokens.IERC721ABI, "")
 	IERC1155 = artifact("IERC1155", tokens.IERC1155ABI, "")
 
+	NiftyswapExchange = artifact("NIFTYSWAP_EXCHANGE", contracts.NiftyswapExchangeABI, contracts.NiftyswapFactoryBin)
```

Comment:
@pkieltyka I'm also unsure about this part, I tried to make it look similar to the others, does it look correct? Should we also use or add `INifty*` stuff?